integration/test_dataset.py

```diff
@@ -56,6 +56,62 @@ def test_replace_values(self):
         assert r[variable.body["id"]] == [1, 2, 3, 4, 5]
         ds.delete()
 
+    def test_append_dataset(self):
+        ds = site.datasets.create(as_entity({"name": "test_scrunch_append_dataset"})).refresh()
+        ds.variables.create(
+            as_entity(
+                {
+                    "name": "my_var",
+                    "alias": "my_var",
+                    "type": "numeric",
+                    "values": [1, 2, 3, None, 5],
+                }
+            )
+        ).refresh()
+        ds_to_append = site.datasets.create(as_entity({"name": "test_scrunch_dataset_to_append"})).refresh()
+        datetime_var = ds_to_append.variables.create(
+            as_entity(
+                {
+                    "name": "my_datetime_var",
+                    "alias": "my_datetime_var",
+                    "type": "datetime",
+                    "resolution": "ms",
+                    "values": [
+                        "2024-01-03T20:00:52.333",
+                        "2024-02-03T20:00:52.234",
+                        "2024-03-03T20:00:52.456",
+                        "2024-04-03T20:00:52.999",
+                        "2024-06-03T20:00:52.123",
+                    ],
+                }
+            )
+        ).refresh()
+        scrunch_dataset = get_mutable_dataset(ds.body.id, site)
+        scrunch_dataset_to_append = get_mutable_dataset(ds_to_append.body.id, site)
+        # This is intended to leave only two records. Changing the variable `datetime_var`
+        # above also changes the test's results
+        filter_value = "2024-03-15T00:00:00.393"
+        resp = scrunch_dataset.append_dataset(
+            scrunch_dataset_to_append,
+            filter=f"my_datetime_var > '{filter_value}'"
+        )
+        if resp is not None:
+            resp['body']['filter'] = {
+                'args': [
+                    {
+                        'variable': datetime_var['self']
+                    }, {
+                        'value': filter_value
+                    }
+                ],
+                'function': '>'
+            }
+        else:
+            pass
```

Comment:
What does it mean if `resp` is None?

Reply:
mmh, I just reused the test pattern I found in another one. I can just remove it. Thanks